### ISSUE TITLE
Switch to Widgets from QML

### ISSUE THREAD

**Issue by rerpha** [ess-dmsc/nexus-constructor#300]
branch for widgets change: `switch_to_widgets`

Umbrella ticket, this should close the following tickets: 

#155 
#110 
#119 
#80 
#91 
#227 
#171 
#158 
#218 
#223 
#226 
#225 
#272 
#259 

Double check that it fixes all of these tickets before closing this ticket. The milestone can also then be finished. 

Close this ticket when `switch_to_widgets` is merged into master.
But first we should:
- Do a full manual test of the interface
- Update the manual interface test script
- Check all QML and related code has been removed from the codebase

**Comment by rerpha** [comment]
All of those tickets have been closed
UI tests have now been added so manual tests can most likely be completely eliminated when #404 is unblocked. 
No more QML left to remove - All functionality has been implemented besides the bits we don't need (internal object JSON view, transforms in add component window) 

We are just waiting now for transformations to be complete and then the code can be merged into master and this issue closed.
